devices/XsensHumanStateProvider/XsensHumanStateProvider.h

```diff
@@ -62,6 +62,8 @@ class hde::devices::XsensHumanStateProvider final
     std::array<double, 3> getBasePosition() const override;
     std::array<double, 4> getBaseOrientation() const override;
     std::array<double, 6> getBaseVelocity() const override;
+    std::array<double, 3> getCoMPosition() const override;
```

Comment:
@kouroshD I do not understand why you are modifying `XsensHumanStateProvider`. If I am not mistaken this is an old implementation that we had while we tried to use the human joint angles from the xsens suit. We do not use this any longer. @lrapetti please confirm this.

Reply:
Because currently, as you can see in [this line](https://github.com/robotology/human-dynamics-estimation/blob/3233bc3c3070dd389329c685df183625d4557fe1/devices/CMakeLists.txt#L9), we currently build the `XsensHumanStateProvider`, which is an implementation of the abstract class `IHumanState`. So, if we do not need the `XsensHumanStateProvider` we can eliminate its corresponding folder (also to avoid confusion) and removes it from this [CMakeLists.txt](https://github.com/robotology/human-dynamics-estimation/blob/integration_based_teleoperation/devices/CMakeLists.txt) as well. Otherwise, it returns an error of `unimplemented pure virtual method`.